code/modules/antagonists/pirate/pirate_gangs.dm

```diff
@@ -115,3 +115,38 @@ GLOBAL_LIST_INIT(heavy_pirate_gangs, init_pirate_gangs(is_heavy = TRUE))
 	response_received = "Thanks for the credits, landlubbers."
 	response_too_late = "Too late to beg for mercy!"
 	response_not_enough = "Trying to cheat us? You'll regret this!"
+
+///Expirienced formed employes of Interdyne Pharmaceutics now in a path of thievery and reckoning
+/datum/pirate_gang/interdyne
+	name = "Expharmacist Unrest"
+
+	is_heavy_threat = TRUE
+	ship_template_id = "interdyne"
+	ship_name_pool = "interdyne_names" //just points to THE ONE AND ONLY
+
+	threat_title = "Funding for Research"
+	threat_content = "Greetings, this is the %SHIPNAME. We require some funding for our pharmaceutical operations. \
+		%PAYOFF credits should suffice."
+	arrival_announcement = "We humbly ask for a substantial amount of income for the future research of our cause. It sure would be a shame if you got sick, but we can fix that."
+	possible_answers = list("Very well.","Get a job!")
+
+	response_received = "Thank you for your generosity. Your money will not be wasted."
+	response_too_late = "We hope you like skin cancer!"
+	response_not_enough = "This is not nearly enough for our operations. I'm afraid we'll have to borrow some."
+
+///Previous Nanotrasen Assitant workers fired for many reasons now looking for revenge and your bank account.
+/datum/pirate_gang/grey
+	name = "The Grey Tide"
+
+	ship_template_id = "grey"
+	ship_name_pool = "grey_names" //just points to THE ONE AND ONLY
+
+	threat_title = "This is a Robbery"
+	threat_content = "Hey it's %SHIPNAME. Give us money. \
+		%PAYOFF might be enough."
+	arrival_announcement = "Nice stuff you got there, it's ours now."
+	possible_answers = list("Please don't hurt me.","GO AWAY GREY SHIT!!.")
+
+	response_received = "Wait, you ACTUALLY gave us the money? Thanks, but we're coming for the rest anyways!"
```

Comment:
Doesn't `response_received` mean they're not coming aboard at all?

Reply:
its flavor tiders dont care about the money its a comedic sentence